app/metrics.yaml

```diff
@@ -205,11 +205,10 @@ events:
         description: |
           A string containing the name of the item the user tapped. These items
           include:
-
-            Settings, Help, Desktop Site toggle on/off, Find in Page,
-            New Tab, Private Tab, Share, Report Site Issue, Back/Forward button,
-            Reload Button, Quit, Reader Mode On, Reader Mode Off, Open In app,
-            Add To Top Sites, Add-ons Manager, Bookmarks, History
+          add_to_homescreen, add_to_top_sites, addons_manager, back, bookmark, bookmarks, desktop_view_off,
+          desktop_view_on, downloads, find_in_page, forward, help, history, library, new_private_tab, new_tab,
+          open_in_app, open_in_fenix, quit, reader_mode_appearance, reader_mode_off, reader_mode_on, reload,
+          save_to_collection, set_default_browser, settings, share, stop, sync_account, and sync_tabs.
```

Comment:
This actually needs changes. We're hitting the character per line limit of 80 with the glean parser, which is why you see CI builds are failing.

>   ERROR running glean_parser v2.2.0
  Sorry, Glean found some glinter nits:
  /home/runner/work/fenix/fenix/app/metrics.yaml (208:81) - line too long (107 > 80 characters) (line-length)
  /home/runner/work/fenix/fenix/app/metrics.yaml (209:81) - line too long (110 > 80 characters) (line-length)
  /home/runner/work/fenix/fenix/app/metrics.yaml (210:81) - line too long (108 > 80 characters) (line-length)
  /home/runner/work/fenix/fenix/app/metrics.yaml (211:81) - line too long (102 > 80 characters) (line-length)